### PR TITLE
Enable ssl

### PR DESCRIPTION
**Pull request by tuanknbl**
- Replace the package `ts-postgres` with `pg` since `ts-postgres` doesn't support SSL yet
- Add `ssl` to each connection config and add a prompt for user to enable/disable SSL when creating a new connection

### CODE REVIEW

**Comment by benjamcalvin** [comment]
We'll want to add `ssl` to `POSTGRES_PARAMS` too, right?

File: src/connection.ts [hunk 1]
```diff
@@ -173,11 +175,33 @@ export async function addConn() {
                 connection[key] = value
             }
         }
+
+        connection['ssl'] = await sslPicker()
```
Comment (tuanknbl) [inline src/connection.ts]: Use sslPicker prompt when adding a normal connections

File: src/connection.ts [hunk 2]
```diff
@@ -136,7 +137,8 @@ export async function importConnFromDbfacts() {
     connId = '(dbfacts)' + connId
     const params = parseDbFactsConnection(connId)
     const connection = {
-        'type': params['type']
+        'type': params['type'],
+        'ssl': await sslPicker()
```
Comment (tuanknbl) [inline src/connection.ts]: Use sslPicker when importing connection from db-facts. by default db-facts does not have an `ssl` parameter

File: src/connection.ts [hunk 3]
```diff
@@ -95,6 +95,7 @@ export async function getPostgresParams() {
         }
         dbConnParams['password'] = await keytar.getPassword('vscode.vscode-sql', activeConn)
     }
+    dbConnParams['ssl'] = vscode.workspace.getConfiguration(`vscodeSql.connections.${activeConn}`).get('ssl')
```
Comment (tuanknbl) [inline src/connection.ts]: get the `ssl` param directly from user settings for both db-facts and normal connections

**Review by tuanknbl — COMMENTED** [review]
dbfact does not have a parameter for `ssl` so I'd have to handle just this param differently than a normal connection. In addition, the prompt for SSL will be a picker instead of an input box like `host` or `user`. So the for loop over `POSTGRES_PARAM` isn't saving me much code.  You can see below where I handle ssl differently than other params.

I can put `ssl` in `POSTGRES_PARAMS` and move the if statements around if you prefer all the params staying in the same place. It'd be an easy change.

**Comment by benjamcalvin** [comment]
Ah, I was looking on the phone, and didn't see you'd covered it! It does
feel like the SSL parameters would ideally be located in the parameter
thing with special handling for dbfacts, but not a big deal.

**Review by benjamcalvin — APPROVED** [review]

**Pull request merged**